package.json

```diff
@@ -17,13 +17,13 @@
     "build": "hardhat compile",
     "build:factory": "tsc -p tsconfig.json",
     "deploy": "hardhat singleton-deployment --network",
-    "test:js": "hardhat test ./src/*/__tests__/*.spec.ts",
+    "test:js": "npx hardhat test ./src/factory/__tests__/index.spec.ts",
```

Comment:
Why use npx hardhat in these scripts rather than the local instance of hardhat?

Reply:
As I understand it, `npx` prefers locally installed versions of packages. However, I added this while trying to troubleshoot why linting didn't work. The add is unnecessary, so I've removed it from the most recent commit. I've confirmed that everything works without `npx` in place.